biz.aQute.bnd.gradle/src/aQute/bnd/gradle/Resolve.groovy

```diff
@@ -145,6 +155,7 @@ public class Resolve extends DefaultTask {
         throw new GradleException("${run.getPropertiesFile()} resolution exception", e)
       } finally {
         logReport(run, logger)
+        run.setParent(parent)
```

Comment:
Not sure I see the utility in restoring the parent just before the run is closed. I would just avoid this.